pkg/cmd/diagnostics/diagnostics_wait.go

```diff
@@ -0,0 +1,89 @@
+package diagnostics
+
+import (
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/dcos/dcos-cli/api"
+	diagnostics "github.com/dcos/dcos-core-cli/pkg/diagnostics/v2"
+	"github.com/dcos/dcos-core-cli/pkg/pluginutil"
+	"github.com/spf13/cobra"
+)
+
+const pollInterval time.Duration = time.Second
+
+func newDiagnosticsWaitCommand(ctx api.Context) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "wait <bundle-id>",
+		Short: "Wait until the given bundle is completed",
+		Args:  cobra.MaximumNArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			c := pluginutil.HTTPClient("")
+			client := diagnostics.NewClient(c)
+
+			// this seemed to be the easist way to manage using two different methods
+			// of getting the right bundle depending on the arguments
+			bundle, err := func() (*diagnostics.Bundle, error) {
+				if len(args) == 0 {
+					return latestBundle(client)
+				}
+				return client.Get(args[0])
+			}()
+			if err != nil {
+				return err
+			}
+
+			// only print the ID after verifying that the bundle actually exists
+			fmt.Fprintln(ctx.Out(), bundle.ID)
```

Comment:
Can we move this at the end? It would be easier to see when the command is finished if it prints right before exiting.

Reply:
I would rather have it print before so that the user can see if it's actually looking at the bundle they expect. Alternatively, I'd say have it print nothing at all. I don't see much use in telling the user what bundle they were waiting on after they spent multiple minutes waiting for it.

Reply:
Maybe log `Waiting for bundle {id}` with the info level then? Also, how about doing it only when the ID is not explicitly passed?